Makefile

```diff
@@ -126,15 +122,15 @@ export_addon: addon
 zip: addon
 	# FIXME: should remove web-ext-artifacts/*.zip first
```

Comment:
I just hit the bug this FIXME intends to fix (multiple zip files inside web-ext-artifacts causes `mv` to bail) :-P